components/compliance-service/api/tests/export_test.go

```diff
@@ -367,3 +311,147 @@ func TestJSONExportWithTwoControlFiltersReturnsError(t *testing.T) {
 	grpctest.AssertCode(t, codes.InvalidArgument, err)
 	assert.Equal(t, "rpc error: code = InvalidArgument desc = Invalid: Only one 'control' filter is allowed", err.Error())
 }
+
+func TestJSONNodeExportReturnsAllExpectedReportsForNode(t *testing.T) {
+	// get reporting client
+	conn, err := getClientConn()
+	require.NoError(t, err)
+
+	defer conn.Close()
+
+	reporting := rs.NewReportingServiceClient(conn)
+	require.NoError(t, err)
+
+	filterQuery := rs.Query{
+		Type: "json",
+		Filters: []*rs.ListFilter{
+			{Type: "start_time", Values: []string{"2018-02-08T00:00:00Z"}},
+			{Type: "end_time", Values: []string{"2018-04-08T09:18:41Z"}},
+			{Type: "node_id", Values: []string{"9b9f4e51-b049-4b10-9555-10578916e149"}},
+		},
+	}
+
+	stream, err := reporting.ExportNode(context.Background(), &filterQuery)
+	require.NoError(t, err)
+
+	reports, err := getJSONReportsFromStream(stream)
+	require.NoError(t, err)
+
+	assert.Equal(t, 3, len(reports))
+
+	assert.Equal(t, "bb93e1b2-36d6-439e-ac70-cccccccccc07", reports[0].GetId())
+	endTime := ptypes.TimestampString(reports[0].GetEndTime())
+	require.NoError(t, err)
```

Comment:
This test keeps testing `err` with no new `err` assigns... ?

Reply:
oh heh. ya i was using a diff call that returned an error before. ill remove those